app/models/manageiq/providers/autosde/storage_manager/cloud_volume.rb

```diff
@@ -100,20 +112,22 @@ def params_for_update
   end
 
   def self.params_for_create(provider)
-    services = provider.storage_services.map { |service| {:value => service.id.to_s, :label => service.name} }
+    capabilities = []
+    provider.capabilities.map do |capability|
+      capabilities.push({:label => "#{capability['abstract_capability']}: #{capability['value']}", :value => capability['uuid']})
+    end
```

Comment:
You already are using `.map` so might as well just assign the result
```suggestion
    capabilities = provider.capabilities.map do |capability|
      {:label => "#{capability['abstract_capability']}: #{capability['value']}", :value => capability['uuid']}
    end
```